sdk/download.go

```diff
@@ -318,3 +319,18 @@ func UpdateSignedFile(file, url string, client *http.Client) error {
 	plog.Infof("Verified file: %s", file)
 	return nil
 }
+
+func DetectSDKVersion(url, branch string) (string, error) {
```

Comment:
After https://github.com/coreos/mantle/pull/220/files#r48512571 you can call GetSDKVersion to re-use an existing repo clone instead of fetching.